### PR TITLE
Esbuild by @valexr

### PR DESCRIPTION
**Pull request by blokhin**

### CODE REVIEW

**Comment by blokhin** [comment]
@Valexr why `app.config.js` has been deleted?

**Comment by blokhin** [comment]
@Valexr also, please, could you make the dog robot happy?

File: package.json [hunk 1]
```diff
@@ -1,99 +1,106 @@
...
-    "optimade-mpds-nlp": "^0.1.1",
-    "sirv-cli": "^1.0.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/tilde-lab/optimade.science.git"
-  }
+	"name": "optimade-esbuild",
+	"version": "1.0.0",
+	"description": "Starter for new Svelte application with ESBuild bundler ",
```
Comment (blokhin) [inline package.json:4]: @Valexr why the name, version and descr have been changed? The version should probably be `2.0.0`.

File: src/stores/media.ts [hunk 2]
```diff
@@ -15,4 +23,43 @@ const mediaqueries = {
...
+
+        for (const query in queries) {
+            const media = window.matchMedia(queries[query] as string)
+            setMatches(media, query)
+            media.onchange = (e) => setMatches(e, query)
+        }
+
+        subscribe((match): void => persist(storage).setItem(storage.key as string, JSON.stringify(match)));
+
+        function setMatches(media: MediaQueryList | MediaQueryListEvent, query: string) {
```
Comment (blokhin) [inline src/stores/media.ts:45]: @Valexr github robot says: `Move function declaration to function body root`

**Comment by blokhin** [comment]
@Valexr the conflicts should be resolved

**Comment by Valexr** [comment]
> @Valexr why `app.config.js` has been deleted?

It's not needed.

**Pull request merged**